A42-xds-ring-hash-lb-policy.md

```diff
@@ -361,13 +365,35 @@ The `ring_hash_experimental` policy will have the following config:
 
 ```
 message RingHashLoadBalancingConfig {
+  // A client-side option will cap these values to 4096.  If either of these
+  // values are greater than the client-side cap, they will be treated
+  // as the client-side cap value.
   uint64 min_ring_size = 1;  // Optional, defaults to 1024.
-  uint64 max_ring_size = 2;  // Optional, defaults to 8M.
+  uint64 max_ring_size = 2;  // Optional, defaults to 4096, max is 8M.
```

Comment:
All the "8M" stuff is pretty confusing, because it isn't the max from this perspective. The max can change. It seems like we should say it will be limited to 4096 without client opt-in here and elsewhere. And then for the configuration to change the max we say it will be limited to 8M.

Reply:
Are you suggesting that we not impose the 8M cap in the gRPC LB policy, only in the xDS code?  Currently, we impose this cap in both places, rejecting any value greater than that.

I suppose that now that we have the client-side cap, there isn't technically any reason we need the gRPC LB policy to reject a config that sets this to a value greater than 8M.  But we also don't currently have the ability to use this policy without xDS, so it's mostly a moot point anyway.